config/profile_stub.config

```
@@ -22,6 +22,9 @@ params {
   ref_json = "test/stub-refs.json"
   ref_rootdir = 'test/references'
 
+  // turn on cell typing and CNV inference
+  perform_cnv_inference = true // this will set cell typing to true as well
+
```

I think better to be explicit
```suggestion
  perform_celltyping = true
  perform_cnv_inference = true 
  
```